server/handlers/validation_test.go

```diff
@@ -19,21 +20,16 @@ import (
 	"github.com/docker/notary/server/storage"
 )
 
-func copyTimestampKey(t *testing.T, fromKeyDB *keys.KeyDB,
-	toStore storage.MetaStore, gun string) {
-
-	role := fromKeyDB.GetRole(data.CanonicalTimestampRole)
-	assert.NotNil(t, role, "No timestamp role in the KeyDB")
-	assert.Len(t, role.KeyIDs, 1, fmt.Sprintf(
-		"Expected 1 timestamp key in timestamp role, got %d", len(role.KeyIDs)))
-
-	pubTimestampKey := fromKeyDB.GetKey(role.KeyIDs[0])
-	assert.NotNil(t, pubTimestampKey,
-		"Timestamp key specified by KeyDB role not in KeysDB")
-
-	err := toStore.SetKey(gun, data.CanonicalTimestampRole, pubTimestampKey.Algorithm(),
-		pubTimestampKey.Public())
-	assert.NoError(t, err)
+func copyKeys(t *testing.T, from signed.CryptoService, roles ...string) signed.CryptoService {
+	memKeyStore := trustmanager.NewKeyMemoryStore(passphrase.ConstantRetriever("pass"))
+	for _, role := range roles {
+		for _, keyID := range from.ListKeys(role) {
+			key, _, err := from.GetPrivateKey(keyID)
+			assert.NoError(t, err)
+			memKeyStore.AddKey(path.Base(keyID), data.CanonicalTimestampRole, key)
```

Comment:
Oh oops, sorry, that was left over from before when I was trying to copy just the timestamp key.  On the server it looks like we just get the ID without caring about the role, so the tests don't seem to be breaking, but I should definitely change this, thanks!